bundles/org.openhab.binding.salus/src/main/java/org/openhab/binding/salus/internal/handler/CloudBridgeHandler.java

```diff
@@ -0,0 +1,263 @@
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        // no commands in this bridge
+        logger.debug("Bridge does not support any commands to any channels. channelUID={}, command={}", channelUID,
+                command);
+    }
+
+    @Override
+    public void dispose() {
+        var localScheduledFuture = scheduledFuture;
+        if (localScheduledFuture != null) {
+            localScheduledFuture.cancel(true);
+            scheduledFuture = null;
+        }
+        super.dispose();
+    }
+
+    @Override
+    public SortedSet<@NonNull DeviceProperty<?>> findPropertiesForDevice(String dsn) {
+        return requireNonNullElse(devicePropertiesCache.get(dsn), emptySortedSet());
+    }
+
+    @Nullable
+    private SortedSet<DeviceProperty<?>> loadPropertiesForDevice(String dsn) {
+        var api = salusApi;
+        if (api == null) {
+            logger.error("Cannot find properties for device {} because salusClient is null", dsn);
+            return null;
+        }
+        logger.debug("Finding properties for device {} using salusClient", dsn);
+        var response = api.findDeviceProperties(dsn);
+        if (response.failed()) {
+            logger.error("Cannot find properties for device {} using salusClient\n{}", dsn, response.error());
+            return null;
+        }
+        return response.body();
+    }
+
+    @Override
+    public boolean setValueForProperty(String dsn, String propertyName, Object value) {
+        var api = salusApi;
+        if (api == null) {
+            logger.error("Cannot set value for property {} on device {} because salusClient is null", propertyName,
+                    dsn);
+            return false;
+        }
+        logger.debug("Setting property {} on device {} to value {} using salusClient", propertyName, dsn, value);
+        var response = api.setValueForProperty(dsn, propertyName, value);
+        if (response.failed()) {
+            logger.error("Cannot set property {} on device {} to value {} using salusClient\n{}", propertyName, dsn,
+                    value, response.error());
+            devicePropertiesCache.invalidate(dsn);
+            return false;
+        }
+        var setValue = response.body();
+        if (setValue != null
+                && (!(setValue instanceof Boolean) && !(setValue instanceof String) && !(setValue instanceof Number))) {
+            logger.warn(
+                    "Cannot set value {} ({}) for property {} on device {} because it is not a Boolean, String, Long or Integer",
+                    setValue, setValue.getClass().getSimpleName(), propertyName, dsn);
+            return false;
+        }
+        var property = devicePropertiesCache.get(dsn).stream().filter(prop -> prop.getName().equals(propertyName))
+                .findFirst();
+        if (property.isEmpty()) {
+            var simpleName = setValue != null ? setValue.getClass().getSimpleName() : "<null>";
+            logger.warn(
```

Comment:
```suggestion
            logger.debug(
```

Reply:
are you sure it should be debug?

Reply:
The cache will be invalidated, but the value will not be changed. So I thought I will log is as warn so the user can backtrack why value of a channel did changed in OH, but not in real device.

Reply:
should I close this (same use case as https://github.com/openhab/openhab-addons/pull/16065#discussion_r1460575819 )?

Reply:
ping

Reply:
This would typically be debug.

Reply:
It is very similar, still think this should be debug, but if you want to have it as warn, leave it as is and we will see how loud this might be in the future.